Extensions/classic_tags.js

```diff
@@ -67,12 +67,21 @@ XKit.extensions.classic_tags = new Object({
 			$(".result_link").each(function() { $(this).attr("target", ""); });
 		}
 		if (XKit.extensions.classic_tags.preferences.redirect_to_tagged.value) {
-			$(".result_link").each(function() { $(this).attr("href", $(this).attr("href").replace("/search/", "/tagged/")); });
-		} else {
-			$(".result_link").each(function() { $(this).attr("href", $(this).attr("href").replace("/tagged/", "/search/")); });
+			$(".result_link").each(this.redirect_to_tagged);
```

Comment:
Not exactly, I get a ``Cannot read property 'call' of undefined``. Debugging the function shows me that this.redirect_to_tagged is indeed undefined